contracts/clients-tests/BlsProvider.test.ts

```diff
@@ -428,6 +439,60 @@ describe("BlsProvider", () => {
       "wait",
     );
   });
+
+  it("should return the connection info for the provider", () => {
+    // Arrange
+    const expectedConnection = regularProvider.connection;
+
+    // Act
+    const connection = blsProvider.connection;
+
+    // Assert
+    expect(connection).to.deep.equal(expectedConnection);
+  });
+
+  it("should return the list of accounts managed by the provider", async () => {
+    // Arrange
+    const expectedAccounts = await regularProvider.listAccounts();
+
+    // Act
+    const accounts = await blsProvider.listAccounts();
+
+    // Assert
+    expect(accounts).to.deep.equal(expectedAccounts);
+  });
+
+  it("should send an rpc request to the provider", async () => {
```

Comment:
Undecided on further changes here and whether we should override/extend (not sure what correct terminology is here) the rpc calls to execute bls logic instead, maybe similar to [how we've done in the QuillProvider](https://github.com/web3well/bls-wallet/blob/4f5b7c75d9cf2baef9e44469f016a89d819df4d2/extension/source/background/AggregatorController.ts#L38-L97)?. The overarching question to answer here is what should the expected behaviour be when a developer uses blsProvider.send()?

Reply:
I would expect this to work similarly to the `QuillProvider`/be BLS Wallet specific when:
- Sending a transaction
- Sending multiple transactions (currently overriding/extending `eth_sendTransaction`, we really should switch to a custom RPC namespace like `blsWallet_sendTransactions`).
- Looking up a receipt from a BLS Wallet txn/bundle

And like a normal provider for other send calls.

We actually can't send/sign normal EOA transactions & messages (ECDSA) as we only have our BLS pub/priv keys. We *may* want to add that in the future if:
- Someone wants to bypass an aggregator and submit straight to chain
- Someone wants to use this provider to write a custom aggregator
- We want to use this in our own aggregator

But I think those are out of scope of this initial implementation and we should focus mostly on use cases where a dApp/script/service want an embedded BLS burner wallet, similar to the wallet @blakecduncan created.

Reply:
Thanks for the thoughts. Makes sense regarding expected behaviour. 

Thinking the same that this work is out of scope for initial implementation. Have raised an issue to look at this in the future (out of scope for current milestone): #424